### PR TITLE
MOBILE-1370: Move deploy to github actions

### PR DESCRIPTION
**Pull request by rlepinski**
### What do these changes do?
Moves deploy to github actions.

### Why are these changes necessary?
Matches our other release flows.

### How did you verify these changes?
Verified the build steps are correct. The actual publish steps might be broken but will resolve during release.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      - "[0-9]+.[0-9]+.[0-9]+*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get Version
+        id: get_version
+        run: echo ::set-output name=VERSION::6.2.0
```
Comment (BrianBatchelder) [inline .github/workflows/release.yaml]:
```suggestion
        run: echo ::set-output name=VERSION::6.2.0

```

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (BrianBatchelder) [inline .github/workflows/release.yaml]:
```suggestion
      - uses: actions/checkout@v2

```

File: .github/workflows/release.yaml (hunk 1 above)
Comment (BrianBatchelder) [reply .github/workflows/release.yaml]: Should "6.2.0" be coming from the commit tag?

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          ./gradlew docs:build
+
+      - name: Build
+        env:
+          UNITY_USERNAME: ${{ secrets.UNITY_USERNAME }}
+          UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
+          UNITY_SERIAL: ${{ secrets.UNITY_SERIAL }}
+        run: ./scripts/docker_run.sh
+
+      - name: Github Release
```
Comment (BrianBatchelder) [inline .github/workflows/release.yaml]:
```suggestion
      - name: Create Github Release
```

File: scripts/deploy_bintray.sh [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+# Upload to bintray
+VERSION=$1
+
+upload() {
+  echo -e "Uploading $1 into ${VERSION}n"
```
Comment (BrianBatchelder) [inline scripts/deploy_bintray.sh]: Aren't `$1` and `${VERSION}` the same here, given line 6?

Comment (rlepinski) [reply scripts/deploy_bintray.sh]: No, the args are scoped per function

`$ ./scripts/deploy_bintray.sh 1.0.0` => Uploading build/urbanairship-1.0.0.unitypackage into 1.0.0

Comment (BrianBatchelder) [reply scripts/deploy_bintray.sh]: Ah, yes. Duh.

**Pull request merged**